### PR TITLE
Fix scripts for multiuser and new bucket name

### PR DESCRIPTION
**Pull request by mberhault**

### CODE REVIEW

File: terraform/aws/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ For multi-user cooperation, please see [Terraform's documentation on remote stat
 1. Have an [AWS](http://aws.amazon.com/) account
 2. [Download terraform](https://terraform.io/downloads.html), *version 0.6.7 or greater*, unzip, and add to your `PATH`.
 3. [Valid AWS credentials file](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html#cli-signup).
-4. [Create an AWS keypair](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:sort=keyName) named `cockroach` and save the file as `~/.ssh/cockroach.pem`. If sharing an account with other users, you may want to customize the key name (eg: `cockroach-<myusername>`) and modify the variable as mentioned in the next section.
+4. [Create an AWS keypair](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:sort=keyName) named `cockroach` and save the file as `~/.ssh/<keyname>.pem`. If sharing an account with other users, you may want to customize the key name (eg: keyname=`cockroach-<myusername>`) and modify the variable as mentioned in the next section.
```
Comment (bdarnell) [inline terraform/aws/README.md]: Can we still give `~/.ssh/cockroach.pem` as the default for people who are not sharing accounts?

**Comment by bdarnell** [comment]
LGTM

File: terraform/aws/README.md (hunk 1 above)
Comment (mberhault) [inline terraform/aws/README.md]: yup. that's still the default in `variables.tf`.

Comment (bdarnell) [inline terraform/aws/README.md]: Then the readme should reflect that instead of only showing `~/.ssh/<keyname>.pem`.

Comment (mberhault) [inline terraform/aws/README.md]: switched back and clarified that the filename should also change if the key name is not `cockroach`

**Pull request merged**